go-app.js

```diff
@@ -25,100 +24,115 @@ go.app = function() {
 
     self.states.add('states:start', function(name) {
         return new MenuState(name, {
-            question: "Choose your destiny:",
+            question: 'Welcome to CTA train tracker.Pick a route:',
 
             choices: [
-                new Choice('states:put', "Put something"),
-                new Choice('states:post', "Post something"),
-                new Choice('states:error', "Cause an error")]
+                new Choice('states:red', 'Red Line'),
+                new Choice('states:blue', 'Blue Line'),
+                new Choice('states:brown', 'Brown Line'),
+                new Choice('states:green', 'Green Line'),
+                new Choice('states:orange', 'Orange Line'),
+                new Choice('states:purple', 'Purple Line'),
+                new Choice('states:pink', 'Pink Line'),
+                // new Choice('states:yellow', 'Yellow Line'),
+                new Choice('states:exit', 'Exit')]
         });
     });
 
-    self.states.add('states:put', function(name) {
-        // When the user has responded, we put their response to
-        // httpbin.org. Once httpbin.org has responded, we tell the interaction
-        // machine to go to 'states:done' next. Instead of just giving it the
-        // states name, we also give it additional options: the method that was
-        // performed, and httpbin.org's echo of the content in the response.
-        return new FreeText(name, {
-            question: 'What would you like to put?',
-
-            next: function(content) {
-                return self
-                    .http.put('http://httpbin.org/put', {
-                        data: {message: content}
-                    })
-                    .then(function(resp) {
-                        return {
-                            name: 'states:done',
-                            creator_opts: {
-                                method: 'put',
-                                echo: resp.data.json.message
-                            }
-                        };
-                    });
-            }
-        });
+    self.states.add('states:red', function(name) {
+        return self
+            .http.get(
+                'http://lapi.transitchicago.com/api/1.0/ttpositions.aspx?key=33305d8dcece4aa58c651c740f88d1e2&rt=red&outputType=JSON', {
+                params: {rt: 'red', key: '33305d8dcece4aa58c651c740f88d1e2', outputType: 'JSON'}
```

Comment:
You don't need the `&key=..&rt=..` in the URL if you're specifying them in the params.